AdyenActions/UI/View Controllers/Voucher/Doku/DokuVoucherView.swift

```diff
@@ -20,11 +20,17 @@ internal final class DokuVoucherView: AbstractVoucherView {
         createLabel(with: model.style.amount, text: model.amount, identifier: "amountLabel")
     }()
 
-    private let model: Model
+    private var model: Model
```

Comment:
because `Model ` is a struct and it has a lazy var that's being accessed in `DokuVoucherView`, and lazy vars makes its enclosing object mutable, hence the `var`.